components/elements/inner/sidebar/CollapsedSidebar.js

```diff
@@ -11,7 +11,7 @@ import 'simplebar/dist/simplebar.min.css'
 
 import SidebarFooterIcon from '@/components/klaudsolcms/dropdown/SidebarFooterIcon';
 
-const CollapsedSidebar = ({sidebarButtons, firstName, lastName, defaultEntityType, router, setCollapse}) => {
+const CollapsedSidebar = ({entityTypeLinks, sidebarButtons, firstName, lastName, defaultEntityType, router, setCollapse}) => {
```

Comment:
the goal is to have a separate grouping between the entity types and the other menus ("Profile", "Settings", etc.), but at the moment I can't make `CSidebarNav` to behave properly. 🫠